<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicalImageTool</class>
 <widget class="QMainWindow" name="MedicalImageTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MedicalImageTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QVTKWidget" name="qvtkWidgetCoronalPlane"/>
      </item>
      <item row="1" column="1">
       <widget class="QVTKWidget" name="qvtkWidget3DView"/>
      </item>
      <item row="0" column="1">
       <widget class="QVTKWidget" name="qvtkWidgetSagitalPlane"/>
      </item>
      <item row="0" column="0">
       <widget class="QVTKWidget" name="qvtkWidgetAxialPlane"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpenImage">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionOpenSTL"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCrosshair"/>
    <addaction name="separator"/>
    <addaction name="actionDistanceMeasure"/>
    <addaction name="separator"/>
    <addaction name="actionScreenshotUpperLeft"/>
    <addaction name="actionScreenshotUpperRight"/>
    <addaction name="actionScreenshotLowerLeft"/>
    <addaction name="actionScreenshotLowerRight"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutProgram"/>
   </widget>
   <widget class="QMenu" name="menuVolumeRenderer">
    <property name="title">
     <string>Renderer</string>
    </property>
    <addaction name="actionGPUVolumeRenderer"/>
   </widget>
   <addaction name="menuOpenImage"/>
   <addaction name="menuEdit"/>
   <addaction name="menuVolumeRenderer"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenImage">
   <property name="icon">
    <iconset resource="MedicalImageTool.qrc">
     <normaloff>:/MedicalImageTool/Resources/fileopen.png</normaloff>:/MedicalImageTool/Resources/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>OpenImage</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="MedicalImageTool.qrc">
     <normaloff>:/MedicalImageTool/Resources/exit.png</normaloff>:/MedicalImageTool/Resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCrosshair">
   <property name="text">
    <string>Crosshair</string>
   </property>
  </action>
  <action name="actionDistanceMeasure">
   <property name="text">
    <string>DistanceMeasure</string>
   </property>
  </action>
  <action name="actionAboutProgram">
   <property name="text">
    <string>AboutProgram</string>
   </property>
  </action>
  <action name="actionGPUVolumeRenderer">
   <property name="text">
    <string>GPUVolumeRenderer</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="MedicalImageTool.qrc">
     <normaloff>:/MedicalImageTool/Resources/filesave.png</normaloff>:/MedicalImageTool/Resources/filesave.png</iconset>
   </property>
   <property name="text">
    <string>SaveAs</string>
   </property>
  </action>
  <action name="actionScreenshotUpperLeft">
   <property name="text">
    <string>ScreenshotUpperLeft</string>
   </property>
  </action>
  <action name="actionScreenshotLowerLeft">
   <property name="text">
    <string>ScreenshotLowerLeft</string>
   </property>
  </action>
  <action name="actionScreenshotUpperRight">
   <property name="text">
    <string>ScreenshotUpperRight</string>
   </property>
  </action>
  <action name="actionScreenshotLowerRight">
   <property name="text">
    <string>ScreenshotLowerRight</string>
   </property>
  </action>
  <action name="actionOpenSTL">
   <property name="text">
    <string>OpenSTL</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MedicalImageTool.qrc"/>
 </resources>
 <connections/>
</ui>
